composer.json

```diff
@@ -5,43 +5,38 @@
     "description": "Braintree payment plugin for Sylius applications.",
     "license": "MIT",
     "require": {
-        "php": "^7.3 || ^8.0",
-        "sylius/sylius": "~1.8.0 || ~1.9.0 || ~1.10.0 || ~1.11.0",
-        "php-http/message-factory": "^1.1",
-        "braintree/braintree_php": "^6.4"
+        "php": "^8.0 || ^8.1",
+        "sylius/sylius": "~1.12.0 || ~1.13.0",
+        "braintree/braintree_php": "^6.4",
+        "symfony/webpack-encore-bundle": "^1.17"
     },
     "require-dev": {
-        "behat/behat": "^3.6.1",
-        "behat/mink-selenium2-driver": "^1.4",
+        "behat/behat": "^3.7",
+        "behat/mink-selenium2-driver": "~1.6.0",
+        "bitbag/coding-standard": "^3.0",
         "dmore/behat-chrome-extension": "^1.3",
         "dmore/chrome-mink-driver": "^2.7",
         "friends-of-behat/mink": "^1.8",
         "friends-of-behat/mink-browserkit-driver": "^1.4",
         "friends-of-behat/mink-debug-extension": "^2.0.0",
         "friends-of-behat/mink-extension": "^2.4",
         "friends-of-behat/page-object-extension": "^0.3",
+        "friends-of-behat/service-container-extension": "^1.0",
         "friends-of-behat/suite-settings-extension": "^1.0",
         "friends-of-behat/symfony-extension": "^2.1",
         "friends-of-behat/variadic-extension": "^1.3",
-        "friends-of-behat/service-container-extension": "^1.0",
-        "friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev",
         "phpspec/phpspec": "^7.0",
         "phpstan/extension-installer": "^1.0",
-        "phpstan/phpstan": "0.12.85",
-        "phpstan/phpstan-doctrine": "0.12.33",
-        "phpstan/phpstan-strict-rules": "^0.12.0",
-        "phpstan/phpstan-webmozart-assert": "0.12.12",
+        "phpstan/phpstan": "^1.4",
+        "phpstan/phpstan-doctrine": "^1.0",
+        "phpstan/phpstan-strict-rules": "^1.0",
+        "phpstan/phpstan-webmozart-assert": "^1.0",
         "phpunit/phpunit": "^9.5",
-        "sensiolabs/security-checker": "^6.0",
-        "sylius-labs/coding-standard": "^4.0",
-        "symfony/browser-kit": "^4.4 || ^5.2",
-        "symfony/debug-bundle": "^4.4 || ^5.2",
-        "symfony/dotenv": "^4.4 || ^5.2",
```

Comment:
Why so many dependencies have been removed?

Reply:
Some are outdated and we don't need them anymore because Symfony 5+ includes them by default. Some have not been deleted but just had their order changed.